Sprint-2/2-mandatory-debug/0.js

```diff
@@ -1,14 +1,21 @@
 // Predict and explain first...
 
-// =============> write your prediction here
+// =============> write your prediction here - undefined as we don't have parameters a and b
 
 function multiply(a, b) {
   console.log(a * b);
 }
 
 console.log(`The result of multiplying 10 and 32 is ${multiply(10, 32)}`);
 
-// =============> write your explanation here
+// =============> write your explanation here - The result of multiplying 10 and 32 is undefined
 
```

Comment:
Can you explain *why* it is undefined? Why does the function above return `undefined`?